docs/release_notes.rst

```diff
@@ -1,2 +1 @@
 .. release-notes:: Release Notes
-    :version: 0.4.0, 0.3.0, 0.2.0, 0.1.0, 0.1.0rc2, 0.1.0rc1
```

Comment:
This will be reverted before merging, needed for fork builds to pass. (it's needed here since there is a reno bug)